cpp/kmeans/dominant-colors/dominant-colors.cpp

```diff
@@ -249,15 +249,12 @@ void dominantColors(std::string PathToImage)
   GetColorBarData(values, colors, cluster, assignments, centroids);
   
   // Show the dominant colors.
-  size_t pos = PathToImage.find_last_of("/\\") + 1;
-  size_t len = PathToImage.rfind(".png") - pos;
-  std::string pathToColorBars = PathToImage.substr(pos, len) + "-colors.png";
-  StackedBar(values, colors, pathToColorBars);
+  StackedBar(values, colors, PathToColorBars);
 }
 
 int main()
 {
-  dominantColors("../../../data/jurassic-park.png");
-  dominantColors("../../../data/the-grand-budapest-hotel.png");
-  dominantColors("../../../data/the-godfather.png");
+  dominantColors("../../../data/jurassic-park.png", "jurassic-park-colors.png");
+  dominantColors("../../../data/the-grand-budapest-hotel.png", "the-grand-budapest-hotel-colors.png");
```

Comment:
```suggestion
  dominantColors("../../../data/the-grand-budapest-hotel.png",
      "the-grand-budapest-hotel-colors.png");
```
Just a quick fix for a too-long line.

Reply:
Done. For future reference, what is the column limit? There are some long lines elsewhere in the file.

Reply:
80 (see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#style-guidelines), but there's not a CI job or anything to check it for this repository, so I'm not surprised there are some long lines here and there.